External/Plugins/ASCompletion/Completion/ASGenerator.cs

```diff
@@ -41,6 +42,10 @@ public class ASGenerator
         static private MemberModel contextMember;
         static private bool firstVar;
 
+        static List<ICompletionListItem> known;
+
+        public static List<ICompletionListItem> KnownList => known;

```

Comment:
@Neverbirth Oh lol. It seems like I have to revert a few things... also in the #947.
The power of C# 6.0 is being wasted...